README.md

```diff
@@ -183,6 +184,9 @@ const { astFromSchema } = require('@autotelic/graphql-schema-tools')
 const federatedSchema = require('./schema')
 
 const document = astFromSchema(federatedSchema, {
-  filterDirectives: ['key', 'external', 'requires', 'provides', 'extends']
+  filterDirectives: ['key', 'external', 'requires', 'provides', 'extends'],
+  filterFields: {
+    Query: ['_service', '_entities']
+  }
```

Comment:
Because the schema is flattened out, we wont ever need to filter deeply nested types correct?

Reply:
Correct ✅